src/wwwroot/services/auth.js

```diff
@@ -168,7 +184,7 @@
     function ensureToken() {
       // It is to avoid have different users in UI and AJAX requests
       // TODO: improve it because when token is removed, the request is done anyway, maybe it is possible to cancel request when there is no token
-      var token = $window.localStorage.getItem('jwtToken');
+      var token = loginSession? loginSession.jwtToken : null;
```

Comment:
In the previous implementation we where using local storage value, but now you are using _in memory_ value, also, there is a TODO note that maybe should be taken into account.

Reply:
Yes, you are right I will check that.